frontend/my-app/src/Pages/LandingPage.tsx

```diff
@@ -69,26 +78,144 @@ const LandingPage: React.FC = () => {
     navigate('/')
   }
 
+  const handleFriendMenu = (): void => {
+    console.log('dummy friend pfp actuation')
+  }
+
   return (
-    <div className="container">
-      <p>Current user: {user.email}</p>
-      <div className="buttons">
-        <button onClick={handleGenerateClick}>Generate</button>
-        <button onClick={handlAccountClick}>Account</button>
-        <button onClick={handleLogoutClick}>Logout</button>
+      <div className="container">
+          <AppBar position="relative"
+          sx={{
+            color: '1a7fc1',
+            alignSelf: 'start',
+            width: 'auto',
+            minWidth: 4 / 5,
+            mb: 'auto',
+            mt: 5,
+            mx: 'auto'
+          }}
+          >
+            <Toolbar>
+              <Button color='secondary' variant='contained' onClick={handleLogoutClick}>Logout</Button>
```

Comment:
NIT: I would prefer if some of the style here was cleaned up. For example:
```
<Button color='secondary' variant='contained' onClick={handleLogoutClick}>Logout</Button>
```
should look like
```
<Button
     color='secondary'
     variant='contained'
     onClick={handleLogoutClick}>
          Logout
</Button>
```